Update storage-troubleshoot-linux-file-connection-problems.md

mabicca opened the pull request:
Adding disclaimer about what is currently supported when using SMB 3.0. 

Corrected examples to list 2.1 so customers do not get confused when trying 3.0 in other distros that will not work.

Red Hat for example, the mount command might work when specifying version 3.0 but that does not add encryption, so using 2.1 is the recommendation from Red Hat as its still experimental for them (this also applies to other distributions)

azurecla commented:
Hi __@SilviaDoomra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mabicca committed:
Update storage-troubleshoot-linux-file-connection-problems.md

Adding disclaimer about what is currently supported when using SMB 3.0. 

Corrected examples to list 2.1 so customers do not get confused when trying 3.0 in other distros that will not work.

Red Hat for example, the mount command might work when specifying version 3.0 but that does not add encryption, so using 2.1 is the recommendation from Red Hat as its still experimental for them (this also applies to other distributions)
